lighthouse-core/config/config.js

```diff
@@ -484,6 +529,10 @@ class Config {
     // Locale is special and comes only from flags/settings/lookupLocale.
     settingsWithFlags.locale = locale;
 
+    if (settingsWithFlags.emulatedUserAgent === true) {
```

Comment:
you know, `locale` _was_ special, now everyone wants their setting to have a bespoke initialization :P

I don't know if there's a better way to do this, but leave a comment?